src/main/dml/fenixedu-academic.dml

```diff
@@ -2921,6 +2921,8 @@ valueType org.fenixedu.academic.domain.accounting.EventTypes as EventTypes {
     }
 
     class student.RegistrationDataByExecutionYear  {
+		Boolean reingression;
```

Comment:
Seems that the semantic you are looking for here would be better supported with primitive boolean (you are turning nulls into false).